checkbox 배경 수정

SayBGM opened the pull request:
- 체크박스 배경이 잘못되어서 수정하였습니다.
- 체크박스 사용 방법 주석으로 적어놓았습니다.

Commit:
checkbox 배경 수정
- 체크박스 사용 방법 주석으로 적어놓았습니다.

Commit:
커서 제거

engolder commented:
input[type=checkbox] {
  display: none;
}
이렇게하면 다른 체크박스에도 적용되니까
.class[type=checkbox]로 하는게 좋을것 같고
.input-checkbox는 지금 label에 class로 다는데 그거보다 .input-checkbox는 checkbox에 달고 .input-checkbox-label을 label에 달면 좋을듯

Commit:
네이밍 변경

 - input[type=checkbox] => .input-checkbox[type=checkbox]
 - label에 달리는 class = input-checkbox-label
